providers/openshift.md

```diff
@@ -136,14 +136,88 @@ regardless of the hypervisor).
    
    Or through the UI if you prefer.
 
+
+### Running with CodeReady Containers
+
+
+If you want to develop with an OpenShift 4 provider, CodeReady Containers (CRC)
+is a great option.  It provides all of benefits as minishift (a simple single
+vm openshift on your development machine) but is an OpenShift v4 cluster.
+
+
+#### Requirements
+
+* [`CodeReady Containers`](https://github.com/code-ready/crc) ([Installation Instructions](https://code-ready.github.io/crc/#installation_gsg))
+* A compatible [Operating System](https://code-ready.github.io/crc/#minimum-system-requirements-operating-system_gsg) and the [Required software packages](https://code-ready.github.io/crc/#required-software-packages_gsg)
+
+#### Quickstart
+
+1. [Download](https://cloud.redhat.com/openshift/install/crc/installer-provisioned) ane extract the latest release of CRC
+
+    ```console
+    tar xfJ crc-linux-amd64.tar.xz
+    ```
+2. Copy the crc binary to a location on your PATH
+
+    ```console
+    sudo cp crc-linux-1.8.0-amd64/crc /usr/local/bin
+    which crc
+    /usr/local/bin/crc
+    ```
+
+3. Setup and start CRC (it will ask for an image pull secret which you can get from [cloud.redhat.com](https://cloud.redhat.com/openshift/install/crc/installer-provisioned))
+
+    ```console
+    crc setup
+    INFO Checking if oc binary is cached
+    ...
+
+    Setup is complete, you can now run 'crc start' to start the OpenShift cluster
+    crc start
+    ...
+    ? Image pull secret [? for help]
+    ```
+
+4. Login to the cluster with `oc`, the command will be printed when `crc start` finishes or you can call `crc console --credentials`
+    ```console
+    crc console --credentials
+    oc login -u kubeadmin -p KUBEADMING_PASSWORD https://api.crc.testing:6443
+    ```
+
+5. Now you can setup the management-infra/management-admin service account for use with ManageIQ
```

Comment:
This section is basically https://gist.github.com/NickLaMuro/e2fac8be87ea0e9f429b6f5d75e02176#file-manageiq-addon without the hawkular bits

Thinking we should move this into the openshift repo instead of in a gist

Reply:
This is fine for now